packages/api/src/controllers/asset.ts

```diff
@@ -740,15 +740,15 @@ export const setupTus = async (objectStoreId: string): Promise<void> => {
 async function createTusServer(objectStoreId: string) {
   const os = await getActiveObjectStore(objectStoreId);
   const s3config = await getObjectStoreS3Config(os);
-  const opts: tus.S3StoreOptions & S3ClientConfig = {
-    ...s3config,
+  const tusServer = new tus.Server({
```

Comment:
Do we need to do any edit to the mock server into asset.test.ts?

Reply:
The mock server is right here, on the functions below!